acceptance-test/user-pathways/happy-path/happy-path.test.js

```diff
@@ -17,139 +17,279 @@ const {
+    async function verifyUser() {
+        await clickSelector(page, CONTINUE_BUTTON);
         await page.$eval(ORGANISATION_INPUT, (element) => {
             element.value = 'Barnardos';
         });
         await page.$eval(EMAIL_INPUT, (element) => {
             element.value = 'test.user@homeoffice.gov.uk';
         });
-        await clickContinueButton(1);
+        await clickSelector(page, CONTINUE_BUTTON);
+        // Bypass user clicking email link - Notify Key will not be set during test runs
+        await navigateTo(page, `http://${APP_CONTAINER_HOST}:${APP_CONTAINER_PORT}/nrm/start?token=skip`);
     }
 
-    it('Happy path - Adult', async() => {
-        try {
-            // start
-            await clickContinueButton(1);
-
-            // who-do-you-work-for
-            await whoDoYouWorkForPage();
-
-            // Bypass user clicking email link - Notify Key will not be set during test runs
-            url = `http://${APP_CONTAINER_HOST}:${APP_CONTAINER_PORT}/nrm/start?token=skip`;
-            await page.goto(url);
-
-            // Hit the url a second time since the first page resolves to an error
-            await page.goto(url);
-            await page.setViewport(VIEWPORT);
-
-            // Run through the skeleton until we reach the Where are you making this report? page
-            await clickContinueButton(1);
-
-            // fr-location
-            await page.waitForSelector(LOCATION_ENGLAND_OPTION);
-            await page.click(LOCATION_ENGLAND_OPTION);
-            await clickContinueButton(1);
-
-            // pv-under-age
-            await page.waitForSelector(PV_UNDER_AGE_NO_OPTION);
-            await page.click(PV_UNDER_AGE_NO_OPTION);
-            await clickContinueButton(1);
-
-            // pv-under-age-at-time-of-exploitation
-            await page.waitForSelector(PV_UNDER_AGE_AT_TIME_OF_EXPLOITATION_NO_OPTION);
-            await page.click(PV_UNDER_AGE_AT_TIME_OF_EXPLOITATION_NO_OPTION);
-            await clickContinueButton(1);
-
-            // what-happened
-            await page.waitForSelector(WHAT_HAPPENED_INPUT);
-            await page.$eval(WHAT_HAPPENED_INPUT, (element) => {
-                element.value = 'Test input regarding details of exploitation';
+    /**
+     * Run a sequence of actions to simulate the completion of the first half
+     * of the NRM form
+     *
+     * page.$eval(param1, param2) has a limitation such that we cannot pass a
+     * variable to param2 (function in the second parameter). This is because
+     * the function is excuted within the browser and cannot recognise any
+     * variables passed to it execept those that exist within the browser.
+     *
+     * @param {string} typeOfPV - type of Potential Victim 'child' or 'adult'
+     * @param {bool} caseReferred - does the Potential Victim was their case
+     * referred?
+     *
+     * @returns {Promise}
+     */
+    async function completeNrmFormPart1(typeOfPV, caseReferred) {
```

Comment:
probably rewrite the name of this function to something like `completeFormUptoX`
Next function `completeFormFromXtoY`

Also maybe explain why these are separate in a comment